stats.go

```diff
@@ -312,6 +327,20 @@ func newPoint(v *view.View, row *view.Row, start, end time.Time) *monitoringpb.P
 	}
 }
 
+func newGaugePoint(v *view.View, row *view.Row, start, end time.Time) *monitoringpb.Point {
+	gaugeTime := &timestamp.Timestamp{
+		Seconds: end.Unix(),
+		Nanos:   int32(start.Nanosecond()),
```

Comment:
It seems strange that Seconds uses `end` and Nanos uses `start`. Is this intentional? If so, could you comment on why?

Reply:
oh. it's another bug! I would fix it right now......